src/ucp/core/ucp_context.c

```diff
@@ -2563,6 +2563,31 @@ void ucp_memory_detect_slowpath(ucp_context_h context, const void *address,
     ucs_memory_info_set_host(mem_info);
 }
 
+void ucp_context_memaccess_tl_bitmap(ucp_context_t *context,
```

Comment:
minor, for consistency:
```suggestion
void ucp_context_memaccess_tl_bitmap(ucp_context_h context,
```